feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -75,14 +80,18 @@ export class CustomerListComponent implements OnInit, OnDestroy {
 
   listsEmpty = false;
 
+  enableAsmB2bCustomerList = false;
+
   protected teardown: Subscription = new Subscription();
 
   constructor(
     protected launchDialogService: LaunchDialogService,
     protected breakpointService: BreakpointService,
     protected asmConfig: AsmConfig,
     protected translation: TranslationService,
-    protected asmCustomerListFacade: AsmCustomerListFacade
+    protected asmCustomerListFacade: AsmCustomerListFacade,
+    // TODO: (CXSPA-2722 for remove ) Remove FeatureConfigService for 7.0
+    @Optional() protected featureConfig?: FeatureConfigService
```

Comment:
```suggestion
      constructor(
    launchDialogService: LaunchDialogService,
    breakpointService: BreakpointService,
    asmConfig: AsmConfig,
    translation: TranslationService,
    asmCustomerListFacade: AsmCustomerListFacade,
    // eslint-disable-next-line @typescript-eslint/unified-signatures
    featureConfig?: FeatureConfigService
  );
  /**
   * @deprecated since 7.0
   */
  constructor(
     launchDialogService: LaunchDialogService,
     breakpointService: BreakpointService,
     asmConfig: AsmConfig,
     translation: TranslationService,
     asmCustomerListFacade: AsmCustomerListFacade
  );
  constructor(
    protected launchDialogService: LaunchDialogService,
    protected breakpointService: BreakpointService,
    protected asmConfig: AsmConfig,
    protected translation: TranslationService,
    protected asmCustomerListFacade: AsmCustomerListFacade,
    // TODO: (CXSPA-2722 for remove ) Remove FeatureConfigService for 7.0
    @Optional() protected featureConfig?: FeatureConfigService
  ) {
    this.breakpoint$ = this.getBreakpoint();
  }
```

Reply:
Updated. Please check it again.